configure.ac

```diff
@@ -1,6 +1,6 @@
     AC_INIT([suricata],[6.0.0-dev])
     m4_ifndef([AM_SILENT_RULES], [m4_define([AM_SILENT_RULES],[])])AM_SILENT_RULES([yes])
-    AC_CONFIG_HEADERS([config.h])
+    AC_CONFIG_HEADERS([src/config.h])
```

Comment:
should we rename to `src/autocnf.c` or similar?

Reply:
I'd go the all the way to `src/autoconf.h`.

Reply:
Should we move generated headers into a `gen` style directory? Think we have a build-info.h as well.

Reply:
> Should we move generated headers into a `gen` style directory? Think we have a build-info.h as well.

I don't think so..  Ultimately, to build plugins without the Suricata source installed we need to install the headers to /usr/include/suricata or something like that.  This holds true for libsuricata as well. I'm afraid of this complicating the process until we know more.